mappings/net/minecraft/client/gui/widget/ClickableWidget.mapping

```diff
@@ -1,8 +1,10 @@
-CLASS net/minecraft/class_339 net/minecraft/client/gui/widget/AbstractButtonWidget
+CLASS net/minecraft/class_339 net/minecraft/client/gui/widget/ClickableWidget
+	COMMENT A clickable widget is a GUI element that has many methods to handle different
+	COMMENT mouse actions.
```

Comment:
I'd also note the existence of `message` here since it seems to be something common to all ClickableWidgets.